lib/spree/adyen/form.rb

```diff
@@ -132,7 +132,8 @@ def order_params order
             merchant_reference: order.number.to_s,
             country_code: order.billing_address.country.iso,
             payment_amount: (order.total * 100).to_int,
-            shopper_locale: I18n.locale.to_s.gsub("-", "_")
+            shopper_locale: I18n.locale.to_s.gsub("-", "_"),
+            shopper_email: order.email
```

Comment:
While I was testing this the HMAC Calculation didn't match without it. Below is a screenshot from the "Test HMAC" tab from the skin page. It shows it as a required field.

![screen shot 2016-06-18 at 11 52 15](https://cloud.githubusercontent.com/assets/881759/16170472/ae265fc2-354b-11e6-985c-69815c70f441.png)

Also, the shopper email is also used for velocity fraud checks, so I don't see any harm in having it included either way?